setup.py

```diff
@@ -19,22 +19,16 @@
 sphinx_require = 'Sphinx>=1.5.1'
 
 tests_require = [
-    'check-manifest>=0.25',
-    'coverage>=4.0',
-    'isort>=4.3.3',
-    'pydocstyle>=2.0.0',
```

Comment:
Same as in https://github.com/inveniosoftware/invenio-rdm-records/pull/101 . pydocstyle is still needed.